### PR TITLE
Add a docs note about displayName plugin order

### PR DESCRIPTION
**Pull request by glenjamin**
As discussed in #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # babel-plugin-react-transform
 
-This Babel plugin wraps all React components into arbitrary transforms written by the community.
```
Comment (gaearon) [inline README.md]: Whitespace is significant in Markdown. I put two spaces on purpose to force it to break line.

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Such transforms can do a variety of things:
 * highlight parts of the screen when components update (want to write this one?),
 * etc.
 
-The limit is your imagination and the time you feel compelled to spend on writing these transforms.
```
Comment (gaearon) [inline README.md]: Same here.

File: README.md (hunk 1 above)
Comment (glenjamin) [inline README.md]: Ah sorry, I thought I had my trailing spaces plugin set up to not act on markdown, but I appear to have screwed it up.

The other PR for docs is better anyway!

**Comment by gaearon** [comment]
Nah, I'd appreciate you resubmitting, doesn't hurt to say it twice.

**Comment by gaearon** [comment]
:+1:

**Pull request merged**